src/rtctools_heat_network/influxdb/profile.py

```diff
@@ -40,22 +39,12 @@ def parse_esdl_profiles(es, start_date=None, end_date=None):
             username = None
             password = None
 
-        conn_settings = ConnectionSettings(
-            host=profile_host,
-            port=profile.port,
-            username=username,
-            password=password,
-            database=profile.database,
-            ssl=ssl_setting,
-            verify_ssl=ssl_setting,
-        )
-        time_series_data = InfluxDBProfileManager(conn_settings)
-
-        time_series_data.load_influxdb(
-            '"' + profile.measurement + '"',
-            [profile.field],
-            profile.startDate,
-            profile.endDate,
+        time_series_data = InfluxDBProfileManager.create_esdl_influxdb_profile_manager(
```

Comment:
just checking one thing. Does this create_esdl_influxdb_profile_manager() also replaces the load_influxdb part of the code, or only the creating of the manager?

Reply:
create_esdl_influxdb_profile_manager(): it creates the manager as well as load the profile. The name is a bit ambiguous I think. The cool thing now is that in create_esdl_influxdb_profile_manager() the data is put in the correct format before used as input data in ....load_influxdb(). This prevents future bugs, as data formats etc might change in the future for the load function.

Reply:
check, thnx